README.md

```diff
@@ -21,7 +21,7 @@ For front-end designing I have used Twitter's front-end library Bootstrap4.
 
 ### Features of the quiz:
 
-* Every question is multiple choice question.
+* Every question is multiple-choice questions.
```

Comment:
This makes it worse

Reply:
Correct.

Reply:
Please fix and create another PR. Closing this PR.

Reply:
@akashgiricse Please could you add the `invalid` label too, to make sure this spam doesn't count towards a free Hacktoberfest tshirt?

Reply:
Added. Thanks.